webpack/assets/javascripts/react_app/redux/actions/notifications/index.js

```diff
@@ -1,65 +1,29 @@
 /* eslint-disable promise/prefer-await-to-then */
 import {
-  NOTIFICATIONS_GET_NOTIFICATIONS,
+  NOTIFICATIONS,
   NOTIFICATIONS_TOGGLE_DRAWER,
   NOTIFICATIONS_SET_EXPANDED_GROUP,
   NOTIFICATIONS_MARK_AS_READ,
   NOTIFICATIONS_MARK_GROUP_AS_READ,
   NOTIFICATIONS_MARK_AS_CLEAR,
   NOTIFICATIONS_MARK_GROUP_AS_CLEARED,
-  NOTIFICATIONS_POLLING_STARTED,
   NOTIFICATIONS_LINK_CLICKED,
 } from '../../consts';
-import { doesDocumentHasFocus } from '../../../common/document';
 import { notificationsDrawer as sessionStorage } from '../../../common/sessionStorage';
-import { API } from '../../API';
+import { API, API_OPERATIONS } from '../../API';
+import { stopInterval } from '../../middlewares/IntervalMiddleware';
 
 const defaultNotificationsPollingInterval = 10000;
 const notificationsInterval =
   process.env.NOTIFICATIONS_POLLING || defaultNotificationsPollingInterval;
```

Comment:
I didn't know you could set the default interval with this environment variable.  Should that be added to the markdown?

Reply:
I think this is something related to notification, but I like the idea of adding:
`process.env.DEFAULT_INTERVAL`